src/Microsoft.AspNet.Mvc.ModelBinding/ModelStateDictionary.cs

```diff
@@ -135,7 +135,10 @@ public ICollection<ModelState> Values
         /// <inheritdoc />
         public bool IsValid
         {
-            get { return ValidationState == ModelValidationState.Valid; }
+            get
+            {
+                return ErrorCount == 0;
+            }
```

Comment:
> A model is marked as Unvalidated even if any of its properties are unvalidated which means if a model is skipped from validation, the state for all the properties of that model should be Skipped(assuming such a state existed) instead of Unvalidated, note that the model itself is still validated which means errors at the model level will still be

I think we need to have a whiteboard talk about this because I'm confused. If we're skipping something, why not mark it as valid? at that point we've done all the validation we're going to do for the property.